Makefile

```diff
@@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
 
 # If you're using docker and docker-compose, you can use this Makefile to run
 # commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+# to: DOCKER_PREFIX="docker-compose run --rm web --"
```

Comment:
Can you fix it in `start-docker.sh` instead? Just before `"$@"` on the last
line.

Sent fron phome. Please ecxuse the fact that I can't tyoe.
On Aug 26, 2015 3:04 PM, "wagnerand" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/mozilla/olympia/pull/691#discussion_r38040937>:
>
> > @@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
> >
> >  # If you're using docker and docker-compose, you can use this Makefile to run
> >  # commands in your docker images by setting the DOCKER_PREFIX environment variable
> > -# to: DOCKER_PREFIX="docker-compose run --rm web"
> > +# to: DOCKER_PREFIX="docker-compose run --rm web --"
>
> Ok, should I back out this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/olympia/pull/691/files#r38040937>.
>